test/e2e_node/util.go

```diff
@@ -75,7 +72,11 @@ const (
 )
 
 func getNodeSummary() (*kubeletstatsv1alpha1.Summary, error) {
-	req, err := http.NewRequest("GET", *kubeletAddress+"/stats/summary", nil)
+	kubeletConfig, err := getCurrentKubeletConfig()
+	if err != nil {
+		return nil, fmt.Errorf("failed to get current kubelet config")
+	}
+	req, err := http.NewRequest("GET", fmt.Sprintf("http://%s:%d/stats/summary", kubeletConfig.Address, kubeletConfig.ReadOnlyPort), nil)
```

Comment:
This won't work for IPv6, see IPv6 example:
https://golang.org/src/net/http/request_test.go

Reply:
@bsdnet that's true, but for this scope only ipv4 on 0.0.0.0:10255 address is being used.